recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "pyaxiom" %}
-{% set version = "1.4.2" %}
+{% set version = "1.4.3" %}
 {% set build = 0 %}
-{% set sha256 = "3236c0b2f664d374ae1eae565ef640c18eb3b585403a81fe62a4c4490b4785f0" %}
+{% set sha256 = "62ad14b936f453134c5982b6bd9d4998f0396eb798e4f74ed9d563d793fb00a2" %}
```

Comment:
Ah, that was the key thing I missed, that the source code tarball is being downloaded from github.  That makes sense now.   

And clicking the pencil icon to edit on github gives you the ability to fork and branch automatically, so that should work okay, right?